index.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const swaggerUi = require('swagger-ui-express')
+const swaggerJsdoc = require('swagger-jsdoc')
+const swaggerOptions = require('./swaggerOptions')
+
+const healthCheckRoute = require('./src/routes/health-check/health-check.route')
+const postTestRoute = require('./src/routes/post-test/post-test.route')
+
+const app = express()
+// swagger
+const specs = swaggerJsdoc(swaggerOptions)
+app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(specs))
+
+// Body-parser
+app.use(bodyParser.json())
+
+// App
+app.use(healthCheckRoute)
```

Comment:
You can create an index file in the routes folder for the main route and use it with the prefix /api